azurelinuxagent/distro/default/env.py

```diff
@@ -94,6 +100,22 @@ def handle_dhclient_restart(self):
            self.distro.dhcp_handler.conf_routes()
            self.dhcpid = newpid
 
+     def handle_fstab_update(self):
+        # Only try to mount if the fstab has been modified. This avoids background mounts when admins are working on a unmounted partition
+        # If we find an error, backoff and only check every minute to give time to correct it and avoid filling the log
+        fstabCurrentModifiedStamp = shellutil.run_get_output("stat /etc/fstab -c %Y")
+        if (fstabCurrentModifiedStamp != self.fstabModifiedStamp) and (datetime.datetime.now() > self.lastNotice + datetime.timedelta(seconds=60)):
```

Comment:
parentheses are not necessary.

also isn't `shellutil.run_get_output` returning `ret, output`? It appears like you're saving a `(ret,output)` in `self.fstabCurrentModifiedStamp` and comparing these tuples (pairs) each time? if you use `os.stat()` or `os.path.getmtime()` that problem will disappear.